subsys/ipc/ipc_service/backends/ipc_rpmsg_static_vrings.h

```diff
@@ -130,7 +130,10 @@
  *
  */
 
-#define VDEV_STATUS_SIZE	(4) /* Size of status region */
+/*
+ * Size of the status region (possibly a multiple of the cache line size).
+ */
+#define VDEV_STATUS_SIZE	(32)
```

Comment:
Would it be possible to create some static asserts to make sure `VDEV_STATUS_SIZE`, `CONFIG_IPC_SERVIC_STATIC_VRINGS_ALIGNMENT`, and the `zephyr,buffer_size` dt property are aligned to cache lines?

Reply:
that could be done (maybe in a following PR?).

Reply:
@hubertmis I actually added a commit to this PR.